src/ted2zim/scraper.py

```diff
@@ -722,8 +803,12 @@ def extract_info_from_video_page(self, url, retry_count=0):
                 logger.error(
                     f"Video has not yet been translated into {requested_lang_code}"
                 )
-                return False
-            return self.extract_video_info_from_json(json_data)
+                return False, None
+            # Serialize the data at json_data["playerData"] into a dict
+            # and overwrite it accordingly
+            player_data = json.loads(json_data["playerData"])
+            json_data["playerData"] = player_data
```

Comment:
One-liner is sufficient:

```suggestion
            json_data["playerData"] = json.loads(json_data["playerData"])
```